clifford/_mvarray.py

```diff
@@ -1,57 +1,96 @@
-import numpy as np
+import functools
+import operator
+from typing import Tuple
 
+import numpy as np
 
 from clifford.io import write_ga_file, read_ga_file  # noqa: F401
+from ._layout import Layout
 from ._multivector import MultiVector
 
 dual_array = np.vectorize(MultiVector.dual)
 normal_array = np.vectorize(MultiVector.normal)
 call_array = np.vectorize(MultiVector.__call__)
 
 
+def _interrogate_nested_mvs(input_array) -> Tuple[Tuple[int], Layout, np.dtype]:
```

Comment:
Whoops, had this annotation wrong:
```suggestion
def _interrogate_nested_mvs(input_array) -> Tuple[Tuple[int, ...], Layout, np.dtype]:
```